src/libs/actions/Policy.js

```diff
@@ -388,7 +388,7 @@ function addMembersToWorkspace(invitedEmailsToAccountIDs, welcomeNote, policyID,
     API.write(
         'AddMembersToWorkspace',
         {
-            employees: JSON.stringify(_.map(logins, (login) => ({email: login}))),
+            employees: JSON.stringify(_.map(logins, (login) => ({accountID: login}))),
```

Comment:
I think we're going to hold off on this, I think we want to be able to continue to send emails here because there's no way to invite new members to workspaces by accountID, so the user would have to have known the email - and therefore there's no _problem_ having the email sent here.

Therefore we don't have to modify the backend to accept accountIDs, which will hopefully let us move forward with this faster :D